conf/log4j2.xml.template

```diff
@@ -32,14 +32,26 @@
               -->
             <PatternLayout pattern="%d{yy/MM/dd HH:mm:ss,SSS} %p [%t] %c{1}: %m%n%ex"/>
         </Console>
+        <RollingRandomAccessFile name="file" fileName="${env:CELEBORN_LOG_DIR}/celeborn.log" filePattern="${env:CELEBORN_LOG_DIR}/celeborn.log.%d" ignoreExceptions="false" append="true">
```

Comment:
if no strong reason, please remove `ignoreExceptions` and `append` and use the default value.